src/applications/personalization/dashboard/components/benefit-application-drafts/ApplicationInProgress.jsx

```diff
@@ -6,6 +6,8 @@ import { recordDashboardClick } from '~/applications/personalization/dashboard/h
 
 import DashboardWidgetWrapper from '../DashboardWidgetWrapper';
 
+const formatTitle = (title = '') => capitalize(title).replace(/\bva\b/, 'VA');
```

Comment:
should we move this to `utils/helpers.js`?

Reply:
sure :) we could if you think there's other use-cases that might need something similar?

Reply:
i'm going to think on this for a future ticket and merge PR first so we can test on staging. I'm thinking maybe a generic "ensure VA remains capitalized in string" helper util could also be useful as well 🤔